README.md

```diff
@@ -46,8 +62,140 @@ The main bot will handle all inputs and ouputs from the user.
 
 See in-depth documentation [here](app/greetings/README.md).
 
-## Product & Store Mini-bot
+## Product & Store Mini-bots
 
 This mini-bot will handle any question related to product price, stock, and store information. The bot first determines what topic is of interest (product or store information) and breaks the user's message down into keywords. The bot then compares these words and checks them in order to create the most appropriate response, and returns the response to the main bot.
```

Comment:
add nutrition, so "...any question related to product price, stock, nutrition, and store information."